CMakeLists.txt

```diff
@@ -115,7 +115,7 @@ if (CPP11_FLAG AND FMT_PEDANTIC)
 endif ()
 
 set_target_properties(cppformat
-  PROPERTIES COMPILE_FLAGS "${FMT_EXTRA_COMPILE_FLAGS}")
+  PROPERTIES COMPILE_FLAGS "-DFMT_EXPORTS ${FMT_EXTRA_COMPILE_FLAGS}")
```

Comment:
`FMT_EXPORTS` shouldn't be defined unconditionally, only when the `BUILD_SHARED_LIBS` CMake variable is set, because it breaks building the static library.